packages/beacon-node/src/network/processor/gossipHandlers.ts

```diff
@@ -649,9 +654,14 @@ function getSequentialHandlers(modules: ValidatorFnsModules, options: GossipHand
       try {
         const insertOutcome = chain.inclusionListPool.add(inclusionList);
         metrics?.opPool.inclusionListPoolInsertOutcome.inc({insertOutcome});
+        metrics?.eip7805.inclusionListTransactionsSeen.inc(
+          {source: InclusionListSource.gossip},
+          chain.inclusionListPool.getTransactions(inclusionList.message.slot, metrics).length
```

Comment:
same here, why do we get all of the transactions from pool? it should just be the transaction included in the inclusion list we received